discussion/app/discussion/api/DiscussionApi.scala

```diff
@@ -195,11 +199,15 @@ trait DiscussionApiLike extends Http with ExecutionContexts with Logging {
   }
 
   def postAbuseReport(abuseReport: DiscussionAbuseReport, cookie: Option[Cookie]): Future[WSResponse] = {
-    val url = s"${apiRoot}/comment/${abuseReport.commentId}/reportAbuse"
-    val headers = Seq("D2-X-UID" -> conf.Configuration.discussion.d2Uid, guClientHeader)
-    if (cookie.isDefined) { headers :+  ("Cookie"->s"SC_GU_U=${cookie.get}") }
-    wsClient.url(url).withHeaders(headers: _*).withRequestTimeout(2.seconds).post(abuseReportToMap(abuseReport))
-
+    if(DiscussionStopTrafficFromDotcom.isSwitchedOn) {
+      throw ServiceUnavailableException("Request from Dotcom to Discussion are currently unavailable")
```

Comment:
It is also be possible to handle duplication by creating something like that:
```
private def failIfDisabled: Future[Unit] = if(switchIsOn) Future.failed(...) else Future.successful(())
```

and then do:
```
failIfDisabled.flatMap(_ => super.getJsonOrError(url, onError, headers :+ guClientHeader: _*))
...
```

What do you think?